cluster-controller/src/test/java/io/strimzi/controller/cluster/model/KafkaClusterTest.java

```diff
@@ -101,6 +105,19 @@ private void checkStatefulSet(StatefulSet ss) {
         assertEquals(new Integer(healthDelay), ss.getSpec().getTemplate().getSpec().getContainers().get(0).getLivenessProbe().getInitialDelaySeconds());
         assertEquals(new Integer(healthTimeout), ss.getSpec().getTemplate().getSpec().getContainers().get(0).getReadinessProbe().getTimeoutSeconds());
         assertEquals(new Integer(healthDelay), ss.getSpec().getTemplate().getSpec().getContainers().get(0).getReadinessProbe().getInitialDelaySeconds());
+        assertEquals("foo=bar\n", findEnvVar(ss, KafkaCluster.KEY_KAFKA_USER_CONFIGURATION));
+    }
+
+    private String findEnvVar(StatefulSet ss, String envVarName) {
```

Comment:
`AbstractModel.containerEnvVars(ss.getSpec().getTemplate().getSpec().getContainers().get(0)).get(envVarName)`

Reply:
Good catch ... didn't realised this "synergy". Fixed.